Assets/Scripts/Gameplay/Equipables/LightingStates/Flickering.cs

```diff
@@ -0,0 +1,35 @@
+using UnityEngine;
+using System.Collections;
+
+namespace ProjectWendigo.LightingStates
+{
+    public class Flickering : AState<LightingStateContext>
+    {
+        private float _timer;
+        private Light _lightComp;
```

Comment:
Actually, you use this in each state, you can probably move it in the state context :)

Reply:
Which genius has brought this to your attention?

Reply:
You? 😳